lib/protocol/src/openctx-protocol.schema.ts

```diff
@@ -37,13 +40,17 @@ export interface ResponseError {
     message: string
     data?: unknown
 }
-export interface CapabilitiesResult {
+export interface CapabilitiesResult<K extends ItemKindID = ItemKindID> {
```

Comment:
Does the JSON schema generator understand this stuff? How does this sort of stuff impact the HTTP protocol?

Reply:
That file you edited is auto-generated from the JSON Schema. Don't make edits to this file directly.

Reply:
Lol, we thought it was the other way around. I'll update the JSON schema. I don't expect it to putout any TS generics magic. @keegancsmith